### PR TITLE
[docs] EAS Build: Improve creating your first build docs

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Fixes ENG-5385

# How

This PR improves the existing documentation by following consistent grammar and verbiage. It also uses `Terminal` component for all shell commands to make them copy/pastable. 

# Test Plan

The changes have been tested by running docs locally.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/build/setup.md [hunk 1]
```diff
@@ -4,122 +4,166 @@ title: Creating your first build
...
+<Terminal cmd={['$ npx create-expo-app my-app']} />
+
+EAS Build also works well with projects created by `npx create-react-native-app`, `npx react-native`, `ignite-cli`, and other project bootstrapping tools.
 
 </Collapsible>
 
-<Collapsible summary="🙋 An Expo user account">
+<Collapsible summary="An Expo user account">
+
+EAS Build is available to everybody with an Expo account. You can sign up at [https://expo.dev/signup](https://expo.dev/signup). It is available for both free and paid subscriptions.
```
Comment (jonsamp) [inline docs/pages/build/setup.md:37]:
```suggestion
EAS Build is available to anyone with an Expo account, regardless of whether you pay for EAS or use our free tier. You can sign up at [https://expo.dev/signup](https://expo.dev/signup).
```

File: docs/pages/build/setup.md [hunk 2]
```diff
@@ -4,122 +4,166 @@ title: Creating your first build
...
+Paid subscribers get quality improvements such as additional build concurrencies, priority access to minimize the time your builds spend queueing, and increased limits on build timeouts. Learn more about different plans and benefits at [EAS pricing](https://expo.dev/pricing).
 
 </Collapsible>
 
 ## 1. Install the latest EAS CLI
 
-EAS CLI is the program that you will use to interact with EAS services from your terminal. Install EAS CLI by running `npm install -g eas-cli`. It will notify you when a new version is available (we encourage you to always stay up to date with the latest version).
+EAS CLI is the command-line app that you will use to interact with EAS services from your terminal. To install it, run the command:
+
+<Terminal cmd={['$ npm install -g expo-cli']} />
```
Comment (jonsamp) [inline docs/pages/build/setup.md:47]:
```suggestion
<Terminal cmd={['$ npm install -g eas-cli']} />
```

File: docs/pages/build/setup.md [hunk 3]
```diff
@@ -4,122 +4,166 @@ title: Creating your first build
...
+
+You can also use the above command to check if a new version of the EAS CLI app is available. We encourage you to always stay up to date with the latest version.
```
Comment (jonsamp) [inline docs/pages/build/setup.md:49]:
```suggestion
You can also use the above command to check if a new version of EAS CLI is available. We encourage you to always stay up to date with the latest version.
```

File: docs/pages/build/setup.md [hunk 4]
```diff
@@ -4,122 +4,166 @@ title: Creating your first build
...
 
 > We recommend using `npm` instead of `yarn` for global package installations. You may alternatively use `npx eas-cli`, just remember to use that instead of `eas` whenever it's called for in the documentation.
 
 ## 2. Log in to your Expo account
 
-If you are already signed in through Expo CLI, you don't need to do anything. Otherwise, log in with `eas login`. You can check whether you're logged in by running `eas whoami`.
+If you are already signed in to Expo account using Expo CLI, you can skip the steps described in this section. If you are not, run the following command to log in:
```
Comment (jonsamp) [inline docs/pages/build/setup.md:55]:
```suggestion
If you are already signed in to an Expo account using Expo CLI, you can skip the steps described in this section. If you are not, run the following command to log in:
```

File: docs/pages/build/setup.md [hunk 5]
```diff
@@ -4,122 +4,166 @@ title: Creating your first build
...
+The easiest way to try out EAS Build is to create a build that you can run on your Android device/emulator or iOS simulator. It's quicker than uploading it to a store, and you don't need store developer membership accounts. If you'd like to try this, read about [creating an installable APK for Android](/build-reference/apk) and [creating a simulator build for iOS](/build-reference/simulators).
 
 ### Build for app stores
 
+Before the build process can start for app stores, you will need to have a store developer account and generate or provide app signing credentials.
+
+Whether you have experience with generating app signing credentials or not, EAS CLI does the heavy lifting. You can opt-in for EAS CLI to handle the generating app signing credentials process. Check out the steps for [Android app signing credentials](#android-app-signing-credentials) or [iOS app signing credentials](#ios-app-signing-credentials) process below for more information.
```
Comment (jonsamp) [inline docs/pages/build/setup.md:87]:
```suggestion
Whether you have experience with generating app signing credentials or not, EAS CLI does the heavy lifting. You can opt-in for EAS CLI to handle the app signing credentials process. Check out the steps for [Android app signing credentials](#android-app-signing-credentials) or [iOS app signing credentials](#ios-app-signing-credentials) process below for more information.
```

**Review by jonsamp — APPROVED** [review]
Thanks for making these improvements. I think we should merge this.

Outside of this PR, we should talk about which tense and which person we use in docs. For instance, should we say "Before you set out to create a build..." or "Before **we** set out to create a build..."?

Also, some sentences here are "You do x", or "You'll do x". 

These questions are higher level, so I think we should merge this PR and add preferences like these to the docs styleguide.

**Comment by amandeepmittal** [comment]
Thanks @jonsamp! I went ahead and converted some of the leftover contractions to their full form for now.

While working on the style guide draft, I mentioned using second person (you) and only using first-person when it is necessary to emphasize the docs as the "Expo team".  I am still working on the draft and trying to find more inconsistencies. Will let you know once I am done with it.

**Pull request merged**